.github/workflows/official-pr.yml

```
@@ -0,0 +1,62 @@
+name: Create official images PR
+
+on:
+  pull_request_target:
+    types:
+      - closed
+
+    paths:
+      - "**/Dockerfile"
+      - "**/architectures"
+      - "**/docker-entrypoint.sh"
+      - "generate-stackbrew-library.sh"
+      - "functions.sh"
+
+jobs:
+  pr:
+    runs-on: ubuntu-latest
+    if: github.repository_owner == 'nodejs' && github.event.pull_request.merged_by != ''
+
+    steps:
+      - name: Checkout the docker-node repo
+        uses: actions/checkout@v2
+        with:
+          path: docker-node
+          fetch-depth: 50
+
+      - name: Checkout the official-images repo
+        uses: actions/checkout@v2
+        with:
+          path: official-images
+          repository: docker-library/official-images
+
+      - name: Generate Stackbrew for diff
+        run: |
+          cd docker-node
+          ./generate-stackbrew-library.sh > ../official-images/library/node
+
+      - name: Create PR in official-images
+        id: create-pr
+        uses: peter-evans/create-pull-request@v3
+        with:
+          token: ${{ secrets.GITHUB_API_TOKEN }}
```

There was a small change in the name due to a GitHub naming issue
```suggestion
          token: ${{ secrets.GH_API_TOKEN}}
```
Feel free to just rebase this into your commit